### PR TITLE
Feature/cplp 552 basic components mui

### PR DESCRIPTION
**Pull request by christianraysono**

### CODE REVIEW

**Comment by fatihayyildiz** [comment]
As far as I can see via Storybook, looks fine to me 👍

**Review by fatihayyildiz — APPROVED** [review]

File: portal/code/cx-portal-shared-components/src/components/basic/IconButton/index.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  size?: 'medium' | 'small'
+  variant?: 'outlined'
+}
+
+export const IconButton = ({
+  size = 'medium',
+  variant,
+  ...props
+}: IconButtonProps) => {
+  const outlinedStyle = variant === 'outlined' ? { border: '2px solid' } : {}
```
Comment (oyo) [inline portal/code/cx-portal-shared-components/src/components/basic/IconButton/index.tsx:16]: Why has this to be hard coded here? If other components are using the variant 'outlined' we have to copy it. Prefer this in a global style or theme

File: portal/code/cx-portal-shared-components/src/components/basic/Input/index.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+          width: '100%',
+        }}
+        error={error}
+        variant="filled"
+      >
+        <InputLabel
+          sx={{
+            position: 'relative',
+            transform: 'none',
+            fontSize: 14,
```
Comment (oyo) [inline portal/code/cx-portal-shared-components/src/components/basic/Input/index.tsx]: use reusable variables

File: portal/code/cx-portal-shared-components/src/components/basic/Input/index.tsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+                  ),
+                }
+              : {}
+          }
+          {...props}
+        />
+        <FormHelperText
+          sx={{
+            marginLeft: 0,
+            fontSize: 12,
```
Comment (oyo) [inline portal/code/cx-portal-shared-components/src/components/basic/Input/index.tsx]: use reusable variables

File: portal/code/cx-portal-shared-components/src/components/basic/Logo/index.tsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+import CXLogoText from '../../../assets/logo/cx-logo-text.svg'
+
+interface LogoProps {
+  variant: 'standard' | 'short' | 'text'
+  altText: string
+}
+
+export const Logo = ({
+  variant = 'standard',
+  altText = 'CatenaX logo',
```
Comment (oyo) [inline portal/code/cx-portal-shared-components/src/components/basic/Logo/index.tsx]: 'Catena-X logo'

File: portal/code/cx-portal-shared-components/src/components/basic/Avatar/index.tsx [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+interface AllAvatarProps extends Omit<AvatarProps, 'ImageComponent'> {
+  altText: string
+  userImage?: string
+  notificationCount?: number
+  isNotificationAlert?: boolean
+}
+
+export const UserAvatar = ({
+  userImage = '',
+  altText = 'CatenaX logo',
```
Comment (oyo) [inline portal/code/cx-portal-shared-components/src/components/basic/Avatar/index.tsx]: 'User avatar'

**Review by oyo — APPROVED** [review]
generally approved but some minor comments

**Review by DianaRenz — APPROVED** [review]

**Pull request merged**